### PR TITLE
Don't send a message if no one's birthday is today.

### PR DESCRIPTION
**Pull request by MeMyselfMirai**
Every day, the Birthday bot always sends a message saying "Happy Birthday to " followed by a list of everyone in CSH whose birthdays are that day.  
This can cause problems if a given day is no one's birthday. 
image

This PR makes the bot not send anything if it determines there are no birthdays today.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -29,6 +29,9 @@ async fn main() {
             });
         }
     }
+    if users.len() == 0 {
+        return;
```
Comment (jabbate19) [inline src/main.rs]: Could you change it to still post a message, such as "It's nobody's birthday today :sad-bidoof:" ? We want to ensure it still runs and doesn't just fail

**Review by jabbate19 — CHANGES_REQUESTED** [review]

**Comment by MeMyselfMirai** [comment]
Considering the number of cases, and the simplicity of each branch, I decided to learn how switch-cases/matches worked in Rust and implemented one here.

File: src/main.rs (hunk 1 above)
Comment (MeMyselfMirai) [reply src/main.rs]: "Happy Birthday to no one. :sad-bidoof:"

File: src/main.rs [hunk 2]
```diff
@@ -29,16 +29,17 @@ async fn main() {
...
-            let s = &users[i];
-            output.push_str(&format!("{}, ", s));
+    match users.len() {
+        0 => output.push_str("no one. :sad-bidoof:");
+        1 => output.push_str(&format!("{}!", users[0]));
+        2 => output.push_str(&format!("{} and {}!", users[0], users[1]));
+        user_count => {
+            for i in [..user_count-1] {
+                let s = &users[i];
+                output.push_str(&format!("{:?}, ", s));
```
Comment (jabbate19) [inline src/main.rs]: Why did you turn this into a debug string?

**Review by jabbate19 — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 2 above)
Comment (MeMyselfMirai) [reply src/main.rs]: Uh... what is a debug string? 
Except for replacing `users.len()` with `user_count`, the code should be identical to the current >2 user case.

Comment (MeMyselfMirai) [reply src/main.rs]: After looking at the current master and my branch, it seems that the `{:?}` was left in master, until you removed it with the most recent commit to your master. When I resolved the merge conflict, I re-introduced it.

**Review by jabbate19 — APPROVED** [review]
:shipit:

**Pull request merged**